data_gatherer.py

```diff
@@ -199,10 +213,12 @@ def main(argv):
     # of which stream is slowest. Instead, We should synchronize on all output
     # streams, and we should ensure that even the operators without output
     # streams complete.
+
+    # TODO: Uncomment this section when pulling
     if FLAGS.control == 'simulator_auto_pilot':
         # We insert a synchronizing operator that sends back a command when
         # the low watermark progresses on all input stream.
-        stream_to_sync_on = center_camera_stream
+        # stream_to_sync_on = center_camera_stream
```

Comment:
You can transform this into an if statement so that the data gatherer still works when there's no camera installed.